features/bo_new/dashboard/manage_users.feature

```diff
@@ -0,0 +1,38 @@
+@bo_new @upper_tier @bo_dashboard
+Feature: [RUBY-759] Manage users
+  As a super user
+  I want to amend EA user privileges
+  So that the right people can help our users
+
+Scenario: Agency super user adds an agency user then upgrades them
+  Given I sign into the back office as "agency-super"
+  When I invite a new "agency" user
+  Then the new user has the correct back office permissions
+  When the new user accepts its invitation and setup a password
+  Then the new user is logged in
+  When I sign into the back office as "agency-super"
+    And I update the new user role to an "agency with refund"
```

Comment:
Throughout the tests I've been repeating the naming convention to match the roles in .config.yml . So "agency-refund-payment-user" is more consistent. But I'm happy if you think the plain English version makes more sense.

Reply:
The thing is that I need this value to fit 2 cases: The name of the element (checkbox) and the parsed text on the table to check the correct role has been assigned. Hence, I went for this as is easier to parse it for these 2 cases

Reply:
That's fine, thanks for explaining 👍